repo/scaffold/application.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from ramble.appkit import *
+
+
+class Scaffold(ExecutableApplication):
+    """Scaffold benchmark"""
+
+    name = "scaffold"
+
+    tags = ["python"]
+
+    executable(
+        "modules",
+        "ml load rocm/6.2.1 rocmcc/6.2.1-cce-18.0.1a-magic",
+    )
+    executable(
+        "build",
+        "pip install {package_path} --extra-index-url https://download.pytorch.org/whl/rocm6.2",
```

Comment:
I am using `package_path` as a parameter, since this repository is private. Does there exist a better way for linking this?